bin/client/opts.rs

```diff
@@ -25,6 +25,14 @@ pub struct Opts {
         about = "The directory where bindles are stored/cached, defaults to $HOME/.bindle/bindles"
     )]
     pub bindle_dir: Option<PathBuf>,
+
+    #[clap(
+        short = 'r',
+        long = "keyring",
+        about = "The path to the keyring file. Defaults to $XDG_CONFIG/bindle/keyring.toml"
```

Comment:
It should be XDG_CONFIG/bindle/keyring.toml. There were a few cases where `.bindle` was used because we used to use it (unwrap_or_default() will not join its value... so what you were reading was "XDG_CONFIG/bindle OR .bindle".